hibike/devices/YogiBear/motor.cpp

```diff
@@ -23,16 +27,21 @@ void motorEnable() {
 
 void motorDisable() {
   digitalWrite(enable_pin, LOW);
+  disablePID();
+  resetPID();
+  zeroEncoder();
+  setPWMInput(0);
+  resetDriveMode();
   motorEnabled = false;
 }
 
-bool readMotorEnabled() {
+bool isMotorEnabled() {
   return motorEnabled;
 }
 
 //returns current in amps
 float readCurrent() {
-  return (analogRead(current_pin) - 3.7) / 30.2;
+  return (analogRead(current_pin) / 33.0);
```

Comment:
Done. It is a bit of a magic number which isn't great, but at the moment it is the best we can do